fiatconnect-api.md

```diff
@@ -1717,10 +1727,30 @@ A KYC schema containing personal data about a user, as well as documents such as
 
 The `selfieDocument` and `identificationDocument` fields should be base64 encoded binary blobs representing images.
 
-## 7.3. Initial Entity Support
+### 7.3.2. Fiat Account Schemas
+
+All Fiat Account Schemas supported by FiatConnect MUST contain the `accountName`, `institutionName`, and `fiatAccountType` fields. `accountName` is a friendly, user-definable name for the account.
+`institutionName` is a user-friendly name representing the financial institution/organization the account is with, and `fiatAccountType` is a `FiatAccountTypeEnum` value,
+representing the type of fiat account this schema represents. The `institutionName` and `accountName` fields are required in order for the API to return obfuscated but distinguishable
+account information from the `GET /accounts` endpoint.
+
+####  7.3.2.1. `AccountNumber`
+
+`AccountNumber` is a Fiat Account schema that represents accounts where the only identifying information required is some `accountNumber` string.
+
+```
+{
+	accountName: `string`,
+	institutionName: `string`,
+	accountNumber: `string`,
+	country: `string`
+	fiatAccountType: `FiatAccountTypeEnum.BankAccount`
```

Comment:
What's the relationship between fiatAccountType and fiatAccountSchema? Is this always `BankAccount` for `AccountNumber` schema or can those be different?

Reply:
`AccountNumber` is a schema (potentially one of many) that can be used to transmit information about a Fiat Account of type `BankAccount`, so for `AccountNumber`, this would be fixed, in order to reify the mapping of `AccountNumber` -> `BankAccount`. This could in theory be an implicit mapping, though; we could define it somewhere in the spec by explicitly saying `AccountNumber` represents a Fiat Account of type `BankAccount`.

Reply:
I prefer it here fwiw